lib/api/core/models/repositories/tokenRepository.js

```diff
@@ -147,32 +153,23 @@ class TokenRepository extends Repository {
       if (!decoded._id) {
         throw new jwt.JsonWebTokenError('Invalid token');
       }
-    } catch (err) {
-      let error;
+    }
+    catch (err) {
       if (err instanceof jwt.TokenExpiredError) {
-        error = errorsManager.get('token_expired');
-        error.details = {
```

Comment:
`details` doesn't exist in KuzzleError errors, it is thus not serialized, and so it's useless (also, adding this unexpected property to a class probably triggers a deopt from turbofan)